app/Http/Controllers/HomeController.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+
+class HomeController extends Controller
```

Comment:
Will be better to create the home route returning a view directly instead of creating a Controller.

Example:

```
Route::get('/home', function () {
    return view('home');
});
```

Reply:
Yes in this case it makes more sense.. The idea of creating the controller in the first place was to be consisting in the coding style and later on if some changes will be implemented in the project, the controller is already there ;)
but you are right for this case..